2333/2/libs/rapper.rb

```diff
@@ -0,0 +1,30 @@
+require 'russian_obscenity'
+
+# class Rapper
+class Rapper
+  attr_reader :name, :battles, :number_of_bad_words
+  ROUNDS_ON_BATTLE = 3
+
+  def initialize(name, battles)
+    @name = name
+    @battles = battles
+    @number_of_bad_words = bad_words
+  end
+
+  def bad_words
+    RussianObscenity.dictionary = [:default, './my_dictionary.yml']
```

Comment:
I would move it to the initializer. Seems like we don't need to rewrite it every time we calculate the number of bad words.